browsers/browsers.json

```diff
@@ -0,0 +1,1223 @@
+        "release_date": "2016-09-20",
+        "release_notes": "https://developer.mozilla.org/Firefox/Releases/49",
+        "status": "retired"
+      },
+      "50": {
+        "release_date": "2016-11-15",
+        "release_notes": "https://developer.mozilla.org/Firefox/Releases/50",
+        "status": "retired"
+      },
+      "51": {
+        "release_date": "2017-01-24",
+        "release_notes": "https://developer.mozilla.org/Firefox/Releases/51",
+        "status": "retired"
+      },
+      "52": {
+        "release_date": "2017-03-07",
+        "release_notes": "https://developer.mozilla.org/Firefox/Releases/52",
+        "status": "esr"
+      },
+      "53": {
+        "release_date": "2017-04-19",
+        "release_notes": "https://developer.mozilla.org/Firefox/Releases/53",
+        "status": "retired"
+      },
+      "54": {
+        "release_date": "2017-06-13",
+        "release_notes": "https://developer.mozilla.org/Firefox/Releases/54",
+        "status": "retired"
+      },
+      "55": {
+        "release_date": "2017-08-08",
+        "release_notes": "https://developer.mozilla.org/Firefox/Releases/55",
+        "status": "retired"
+      },
+      "56": {
+        "release_date": "2017-10-03",
+        "release_notes": "https://developer.mozilla.org/Firefox/Releases/56",
+        "status": "current"
+      },
+      "57": {
+        "release_date": "2017-11-28",
+        "release_notes": "https://developer.mozilla.org/Firefox/Releases/57",
+        "status": "beta"
+      },
+      "58": {
+        "release_date": "2018-01-22",
+        "release_notes": "https://developer.mozilla.org/Firefox/Releases/58",
+        "status": "nightly"
+      }
+    }
+  },
+  "ie": {
+    "releases": {
+      "1": {
+        "release_date": "1995-08-16",
+        "status": "retired"
+      },
+      "1.5": {
+        "status": "retired"
+      },
+      "2": {
+        "release_date": "1995-11-22",
+        "status": "retired"
+      },
+      "3": {
+        "release_date": "1996-08-13",
+        "status": "retired"
+      },
+      "4": {
+        "release_date": "1997-09-01",
+        "status": "retired"
+      },
+      "5": {
+        "release_date": "1999-03-18",
+        "status": "retired"
+      },
+      "5.5": {
+        "release_date": "2000-07-01",
+        "status": "retired"
+      },
+      "6": {
+        "release_date": "2001-08-27",
+        "status": "retired"
+      },
+      "7": {
+        "release_date": "2006-10-18",
+        "status": "retired"
+      },
+      "8": {
+        "release_date": "2009-03-19",
+        "status": "retired"
+      },
+      "9": {
+        "release_date": "2011-03-14",
+        "status": "retired"
+      },
+      "10": {
+        "release_date": "2012-10-26",
+        "status": "retired"
+      },
+      "11": {
+        "release_date": "2013-10-17",
+        "status": "current"
+      }
+    }
+  },
+  "opera": {
+    "releases": {
+      "2": {
+        "release_date": "1996-12-01",
```

Comment:
1996-07-14 according this announcement: https://groups.google.com/forum/#!msg/comp.infosystems.www.browsers.ms-windows/XIlBH6cBWoQ/wqsmYKJe1uMJ